### DISCUSSION TITLE
Very impressed, many thanks!

### DISCUSSION THREAD

**Discussion by hagenbuch** [arendst/Tasmota#19661] [General]
Dear Tasmotaoists :)

I am thoroughly impressed and wanted to say thank you:

I am experimenting with Tasmota on ESP8266 since quite some time now, found my way around rules, started a little bit of scripting to try to read infrared interface of electricity counters.

Now, after tinkering here and there, I managed to use the Web Installer and install a fresh Tasmota 13.1.0 (tasmota32solo1.bin) on an Olimex ESP32-PoE that had Microphython on it before.

The surprise points:

- Web installer actually worked even as something else had been flashed before
- Understandably, it just flashed a "vanilla" version but I could just run "Firmware upgrade" to get the latest version!
- It autoconfigured itself to the Olimex ESP32-PoE (wow!) so I guess all ports are set already!
- I can use Ethernet and Wifi as if it would be no big deal
- It shows me the contents not only of the flash storage but also the SDCard. Awesome!
- I just configured GPIO13 and GPIO16 to I2C SDA / SCL
- and before I could even search what to do now to read the sensors, the reading appeared on the website!

Wow. This is much better than I expected.

I've donated 50 EUR already about a week ago but this will not be my last donation. Next step will be to try to integrate two IR interfaces for two counters and / or add a RS485 to read some MODBUS counters!

**Comment by s-hadinger** [comment]
Thanks for your feedback
